toolkit/sidechain/sidechain-slots/src/lib.rs

```diff
@@ -12,10 +16,20 @@ use sidechain_domain::{ScEpochNumber, ScSlotNumber};
 pub use sp_consensus_slots::{Slot, SlotDuration};
 use sp_core::offchain::Timestamp;
 
+/// Number of slots in each Partner Chain epoch
+///
+/// This value should satisfy the following property:
+/// > `slots_per_epoch` * `slot_duration` % `main_chain_epoch_duration` == 0
```

Comment:
PC: 600 slots of 6 seconds is `3600` seconds.
Cardano: 24h = `24*3600`.
`3600 % (24*3600) = 3600`
`(24*3600) % 3600 = 0`

```suggestion
/// > `main_chain_epoch_duration` % (`slots_per_epoch` * `slot_duration`)  == 0
```